configuration/wireless/access-point-bridged.md

```diff
@@ -0,0 +1,163 @@
+
+# Setting up a Raspberry Pi as a bridged wireless access point
+
+The Raspberry Pi can be used as a bridged wireless access point within an existing Ethernet network. This will extend the network to wireless computers and devices.
+
+If you wish to create a standalone wireless network, consider instead setting up a [routed access point.](./access-point-routed.md)
+
+```
+                                         +- RPi -------+
+                                     +---+ Bridge      |
+                                     |   | WiFi AP     +-)))
+                                     |   | 192.168.1.2 |         +- Laptop ----+
+                                     |   +-------------+     (((-+ WiFi STA    |
```

Comment:
Looks like "WiFi STA" is an abbreviation for "WiFi Station", but I'm not sure how commonly-used that is?  "wireless client" would probably work equally as well?
(And similarly, "WiFi AP" would probably work as "wireless access point")

Reply:
As Helen Lyn has suggested - 'wifi' needs to be change to 'wireless' everywhere it occurs.

Reply:
I think in the case of ASCII art, WLAN would be sufficient and the same number of letters.